### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by Mats-SX**

### CODE REVIEW

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+BitNine were originally scheduled for attendance at the oCIM, but had to decline, as the Central European location was unfit given time their time constraints.
+Alastair went through slides by Kisung Kim from BitNine, detailing the hybrid relational/graph architecture of Agens Graph, which is based on PostgreSQL.
+The most highlighted contribution was the way in which BitNine had introduced integration points between SQL and Cypher, allowing each as a subquery construct to the other, and how functions may be shared as expressions between the languages.
+
+---
+
+The lunch break paved way for the big topic previously brought up by Marcus Paradies of SAP in the morning: multiple graphs.
+Cypher has always been a language that operates on a single implicitly understood graph, and producing a stream or records as output (the collection of which effectively form a table).
+Alastair Green discussed motivations for changing this model to make Cypher a language _closed over graphs_, sketching out a future where Cypher would come to operate on a multitude of graphs, and also producing a multitude of graphs as output.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "discussed motivations"
->
"discussed the motivations"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "sketching out" -> "envisioning"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "where Cypher would come to operate on a multitude of graphs, and also producing a multitude of graphs as output."
->
"where Cypher would be capable of processing multiple graphs provided as input, and producing as output one or more graphs."

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+Alastair touched on important subtopics such as identity, addressing, and how compositions of graphs from distinct sources may be defined.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "touched on important"
-> 
"explored salient"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "how compositions of graphs from distinct sources may be defined."
->
"ways of defining compositions of graphs from distinct sources."

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+Stefan Plantikow led a longer session on the very topic, presenting his latest thinking on how Cypher can be remodeled towards a graph-in-graph-out paradigm.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "Following on from Alastair's talk on the vision of multiple graphs in Cypher, Stefan Plantikow led a longer session on the topic"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+Stefan deep-dove into the motivations, but the focus was put on how the concept of multiple graphs could be made logically consistent, and how the execution model could be extended, while still paying respects to Cypher's sizable user base and imposing breaking change in semantics.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "Stefan deep-dove into the motivations, but the focus was put on how the concept of multiple graphs could be made logically consistent, and how the execution model could be extended"
-> 
"Stefan discussed the motivations from a different angle to those covered by Alastair, focusing on how to make the concept of multiple graphs logically consistent and how to extend the execution model"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: ", while still paying respects to Cypher's sizable user base and imposing breaking change in semantics."
-> 
", whilst still keeping in mind Cypher's considerable user base and imposing breaking change in semantics."

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+One of Stefan's main ideas was the re-interpretation of Cypher's result records as graph-records, or 'g-records' (should we retro-actively amend this to mention our new term graphlets?), meaning each binding of a matched subgraph would itself be interpreted as a (typically very small) graph, and the extended ability to collapse/union all such g-records into one large result graph.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "One of Stefan's main ideas"
-> 
"One of the major concepts in Stefan's discussion"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: Re your question about graphlets, maybe mention these parenthetically (i.e. qualify in brackets)

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+Both the g-record model and it's larger sibling would enact Cypher as being _closed over graphs_, as it would be possible to upon retrieval of the result graph(s) immediately issue a new Cypher query, now pattern matching on the newly computed results.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "it's larger sibling"
->
"its larger sibling"

What precisely is the _larger sibling_ - please expand on this

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+Stefan also gave detailed syntax proposals for how to define and use graphs as values in the context of a query, including a take on the addressing topic raised by Alastair.
+It was made clear that this topic is at the top of the minds of several key Cypher innovators.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "at the top of the minds of several"
-> 
"foremost in the minds of several"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+Before Stefan's dive into multi-graph land, Martin Junghanns from the University of Leipzig presented his research project on implementing Cypher on Gradoop, a graph platform based on Apache Hadoop.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "into multi-graph land"
-> 
"into the world of multiple graphs"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: Links to Gradoop and co

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+Martin gave us an overview on how their system handled query planning, and the model used to represent (intermediate) query results in the distributed framework, Flink, that queries are executed in.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "Flink, that queries are executed in."
-> 
"Flink, in which the queries are executed."

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+The project had also featured some interesting extensions to the Property Graph Model, over which Cypher is defined, including a concept of logical subgraphs and a set of graph operations.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "had also featured some interesting"
->
"also featured interesting"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "over which Cypher is defined"
-> 
"upon which Cypher is based"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "including a concept"
->
"including the concept"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+Following Martin's talk, Hannes Voigt from the Technical University of Dresden walked us through his research project, in which also Michael Hunger from Neo had participated.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "Following Martin's"
->
"Following on from Martin's"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "in which also Michael Hunger from Neo had participated"
-> 
"in which Michael Hunger from Neo had also participated"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+The topic was virtual graphs and views, and featured several intriguing extensions to Cypher, which were expressed as able to cross the concept chasm, which Hannes explained as the different levels of abstraction that users view their data in.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "was virtual graphs and views"
->
"comprised virtual graphs and views"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "Cypher, which were expressed as able to cross the concept chasm"
->
"Cypher. These were expressed in terms of 'crossing the concept chasm' "

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+At the lowest level is the pure data, big and raw.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: At the lowest level of abstraction is the actual raw unprocessed data, which is usually very high in volume.

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+At higher levels, larger patterns start to appear, which are made up of groups of nodes and relationships from the lower levels.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "which are made up of"
->
"composed of"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+These larger patterns are in the model constructed using virtual nodes and relationships, with the additional ability to define views which provide several interesting qualities, such as performance optimizations and query modularization.
+
+---
+
+The afternoon session was featured two larger sections in which the members of the Cypher Language Group, Neo's internal team responsible for developing Cypher, presented views and ideas on how to address the most prominently mentioned shortcomings of the language in its current form.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "session was featured two larger sections"
->
"session featured two larger discussions"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+Mats Rydberg presented a proposal for a new schema/constraint syntax, and also talked more in-depth on the Technology Compatibility Kit, highlighting its usefulness to verify that a multitude of language implementations are semantically consistent.
+Petra Selmer went through the latest thinking on several classes of subqueries, including syntax proposals.
+Petra also detailed the revised Cypher improvement process, which has been designed to fit the open, collaborative format intended for the openCypher project.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "designed to fit the"
-> 
"designed to chime in with the"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 17]
```diff
@@ -0,0 +1,107 @@
...
+As mentioned above, Stefan and Oskar discussed semantics of pattern matching, in terms of isomorphism/homomorphism of subgraphs and entities.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: Link to the previous snippet of this discussion (for easier reading)

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "pattern matching, in terms of"
->
"pattern matching in terms of"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 18]
```diff
@@ -0,0 +1,107 @@
...
+Tobias Lindaaker provided insights in how Cypher could complete its support for Conjunctive Regular Path Queries (CRPQs), going through historical thinking as well as recent syntax and semantics proposals.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "insights as to how Cypher"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 19]
```diff
@@ -0,0 +1,107 @@
...
+The topic of vendor-specific extensions, including some of the pitfalls that should be looked out for from experiences with the SQL standard was also presented by Tobias.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "with the SQL standard was also"
->
"with the SQL standard, was also"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 20]
```diff
@@ -0,0 +1,107 @@
...
+
+The last session also featured Paolo Guagliardo from the University of Edinburgh, who gave us a look into a recent research project on formalising semantics of SQL, and what advantages that a language may be given throug a formal specification.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "who gave us a look into a"
->
"who presented a"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "and what advantages that a language may be given throug a formal"
->
"and the advantages conferred on a language through the provision of a formal"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 21]
```diff
@@ -0,0 +1,107 @@
...
+Paolo also announced the newly started project of producing a formal semantics specification of Cypher, which is to be carried out by his research team, including Nadime Francis and prof. Leonid Libkin.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "newly started"
->
"recently begun"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "prof. Leonid Libkin"
->
"Professor Leonid Libkin"

Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: Links to all three people's bio's

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 22]
```diff
@@ -0,0 +1,107 @@
...
+This project will be carried out during the spring of 2017, and reports of progress will be given at upcoming oCIMs.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "will be carried out during"
->
"will be undertaken during"

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 23]
```diff
@@ -0,0 +1,107 @@
...
+
+## Some reflections
+
+Overall, the meeting was received in a very positive light.
```
Comment (petraselmer) [inline _posts/2017-03-23-first-ocim-blog.markdown]: "Overall, the meeting was received in a very positive light."
->
"All in all, the meeting was a resounding success, and we look forward to many more in the future."

File: _posts/2017-03-23-first-ocim-blog.markdown [hunk 24]
```diff
@@ -0,0 +1,107 @@
...
+
+Just before lunch, Alastair Green took the floor again, standing in for BitNine, the Korean company behind Agens Graph, discussing how Cypher, SQL, and other query languages may be integrated with one another.
+BitNine were originally scheduled for attendance at the oCIM, but had to decline, as the Central European location was unfit given time their time constraints.
+Alastair went through slides by Kisung Kim from BitNine, detailing the hybrid relational/graph architecture of Agens Graph, which is based on PostgreSQL.
+The most highlighted contribution was the way in which BitNine had introduced integration points between SQL and Cypher, allowing each as a subquery construct to the other, and how functions may be shared as expressions between the languages.
+
+---
+
+The lunch break paved way for the big topic previously brought up by Marcus Paradies of SAP in the morning: multiple graphs.
+Cypher has always been a language that operates on a single implicitly understood graph, and producing a stream or records as output (the collection of which effectively form a table).
```
Comment (Mats-SX) [inline _posts/2017-03-23-first-ocim-blog.markdown]: It meant to say `of records`

File: _posts/2017-03-23-first-ocim-blog.markdown (hunk 6 above)
Comment (Mats-SX) [inline _posts/2017-03-23-first-ocim-blog.markdown]: Described it as `the companion model of the unionised graph`

**Pull request closed without merging**